canonical-web-and-design/charmhub.io
# Remove hardcoded content from /charms and /bundles pages

## Done

- Remove hardcoded content from /charms and /bundles pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045/charms
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to http://localhost:8045/charms and http://localhost:8045/bundles and see there is only content coming from the API displayed


## Screenshots

![image](https://user-images.githubusercontent.com/40214246/93229727-cf9f2c80-f76e-11ea-8ede-a7c1e8142ddc.png)

![image](https://user-images.githubusercontent.com/40214246/93229768-db8aee80-f76e-11ea-9be2-46bd35cfd067.png)
